catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/SubscriptionsPersistentStoreImpl.java

```diff
@@ -237,6 +237,20 @@ private String quote(String value) {
     return Collections.emptySet();
   }
 
+  @Override
+  public void removeSubscriptions(String id) {
+    final String ecql = queryId(id);
+
+    LOCK.lock();
+    try {
+      persistentStore.delete(SubscriptionsPersistentStore.SUBSCRIPTIONS_TYPE, ecql, 0, 1000);
```

Comment:
✏️ Same thing here

Reply:
Added dependency on PersistentStoreImpl.java and used the constants defined there.